### ISSUE TITLE
Test Suite : Determine how long term we will handle the test suite 

### ISSUE THREAD

**Issue by andorsk** [decentralized-identity/presentation-exchange#395]
@decentralgabe spoke of some test suite that can be used for PE, potentially docker based. Easy to hook in for other implementors.

**Comment by dtmcg** [comment]
Lets try and decompose this into a couple of tasks/issues 
strawman of how we could break this down 
- the test vectors the suite will test 
- the harness mechanism for the suite 
- the client template to be uptake by the suite

**Comment by bumblefudge** [comment]
--> 5 sequential issues
1. one simple test - e2e flow, with one super-simple VC and super-simple `pres_sub`, etc.
2. harness (1 test)
3. normative statement audit
4. rest of tests (could be parallelized) 
5. first 3 gotta be done before

**Comment by brentzundel** [comment]
not sure why this is in milestone v2.1?

**Comment by kimdhamilton** [comment]
> not sure why this is in milestone v2.1?

I suppose I was feeling more optimistic about my bandwidth. Let's make it 3.0

**Comment by nklomp** [comment]
Yeah we had some discussion about it a week ago that the testsuite could also go without a version
